statements.py

```diff
@@ -0,0 +1,46 @@
+#input data
+current_bal: int = 500
+min_withdrawal: int = 20
+max_withdrawal: int = 1000
+
+#Exit function
+def exit():
+    print ("Thank you for choosing our bank! Have a good day!")
+def failure():
+    print ("Transaction failed. Try again")
+
+
+#Transaction function
+def transaction(current_bal):
+    while True:
+        cashamount:int = int(input("Enter cash amount to be withdrawn: "))
+        if cashamount <= current_bal and cashamount <= max_withdrawal and cashamount >= min_withdrawal:
+            current_bal -= cashamount
+            print("Transaction successful. Remaining balance:", current_bal)              
+        elif cashamount > max_withdrawal or cashamount < min_withdrawal:
+            print("This transaction cannot be processed. Withdrawal limits are violated. Your current balance is: ", current_bal)            
+        elif cashamount > current_bal:
+            print("Transaction failed. Insufficient funds. Your current balance is: ", current_bal)            
+        else:
+            break
+        proceed = input("Do you want to proceed? y/n: ")
```

Comment:
when we discuss strings and method, we will learn about the upper and lower method for strings , this can help in making sure all the responses are in the same format.